src/storage/src/client.rs

```diff
@@ -674,15 +754,15 @@ mod v1 {
             assert_eq!(got.domain, from.domain);
             assert_eq!(got.entity_id, from.entity_id);
             assert_eq!(got.etag, from.etag);
-            if let Some(got) = &got.project_team {
-                if let Some(from) = &from.project_team {
-                    assert_eq!(got.project_number, from.project_number);
-                    assert_eq!(got.team, from.team);
-                } else {
-                    panic!("expected None, got {:?}", got); // false, lets get the error.
-                }
+            if let Some(from) = &from.project_team {
+                let got = got
+                    .project_team
+                    .clone()
+                    .expect("project_team should be None");
+                assert_eq!(got.project_number, from.project_number);
+                assert_eq!(got.team, from.team);
```

Comment:
I suspect this can use some simplification, maybe:

```suggestion
            match (&from.project_time, &got.project_team) {
                (None, None) => {}, // yay!
                (Some(_), None) => panic!("expected a value in the project_team, {got:?}"),
                (None, Some(_)) => panic!("unexpected value in the project team, {got:?}"),
                (Some(f), Some(g)) => {assert_eq!( ... ); assert_eq!(... ); }, 
```

Reply:
Done! thanks for the suggestion.